guides/v2.2/cloud/trouble/robots-sitemap.md

```diff
@@ -13,11 +13,11 @@ Please make sure that "/" is writable by the web-server.
 
 With {{site.data.var.ece}}, you can only write to specific directories, such as `var`, `pub/media`, `pub/static`, or `app/etc`. When you generate the `sitemap.xml` file using the Admin panel, you must specify the `/media/` path.
 
-You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the `<domain.your.project>/robots.txt` file.
+You do not have to generate a `robots.txt` because it is generated on demand and stored in the database. It does not create a file, but you can view the content in your browser with the `<domain.your.project>/robots.txt` file.
```

Comment:
Changed to passive?

Reply:
**done**

Reply:
I don't see a change here. Revert.

Reply:
Should I revert this sentence to 
```
You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the `<domain.your.project>/robots.txt` file.
```
?

Thank you!

Reply:
I am sorry for the confusion I caused. I suggest:

```suggestion
You do not have to generate a `robots.txt` file because it generates the `robots.txt` content on demand and stores it in the database. You can view the content in your browser with the `<domain.your.project>/robots.txt` link.
```

Reply:
**fixed**